utils_for_test.go

```diff
@@ -42,6 +42,17 @@ func OutboundConnection(call *OutboundCall) (*Connection, net.Conn) {
 	return conn, conn.conn
 }
 
+// GetConnections returns all connections for a channel.
```

Comment:
You can delete this function (it no longer exists in master, so no need to update it)